### PR TITLE
fix the typo of calculating learning rate in examples_segmentation/voc_sota

### PR DESCRIPTION
**Pull request by hlnull**
The original version of calculating the learning rate in https://gluon-cv.mxnet.io/build/examples_segmentation/voc_sota.html#learning-rates is `lr = base\_lr \times (1-nup/max_nup)^power`, which is inconsistent with the formula in https://mxnet.incubator.apache.org/api/python/optimization/optimization.html#mxnet.lr_scheduler.PolyScheduler. 

In my opinion, the formula in https://mxnet.incubator.apache.org/api/python/optimization/optimization.html#mxnet.lr_scheduler.PolyScheduler shall be right. 
Please correct me if I am wrong.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-761-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-761/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by hlnull** [comment]
@hetong007 
Thanks for comments. Updated.

**Comment by mli** [comment]
Job PR-761-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-761/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by hetong007** [comment]
By looking at the compiled preview site: http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-761/3/build/examples_segmentation/voc_sota.html

You may find that the `power` has a broken format. Please fix this, and check the result in the automatic compiled preview.

**Comment by mli** [comment]
Job PR-761-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-761/4/index.html 
Code coverage of this PR:  vs. Master:

**Comment by hlnull** [comment]
Sorry, forgot to check the changes. Already fixed it.

**Comment by hetong007** [comment]
LGTM.

@zhanghang1989 do you have comments?
